xbmc/input/IRTranslator.h

```diff
@@ -49,6 +50,9 @@ class CIRTranslator
   bool LoadIRMap(const std::string &irMapPath);
   void MapRemote(TiXmlNode *pRemote, const std::string &szDevice);
 
+  // Construction parameters
+  const CProfilesManager &m_profileManager;
```

Comment:
unused if `!(defined(HAS_LIRC) || defined(HAS_IRSERVERSUITE))`

Reply:
"fixed" (still logically unused at runtime, but at least the compiler will see it)